tests/bats/helpers.sh

```diff
@@ -172,3 +172,28 @@ apply_check_delete_workload_imex_chan_inject() {
   kubectl delete -f demo/specs/imex/channel-injection.yaml
   kubectl wait --for=delete pods imex-channel-injection --timeout=10s
 }
+
+
+# Run cmd in nvidia-mig-manager pod because that one has highest privileges. I
+# use this for example to run `nvcnt gb-nvl-027-compute06 nvidia-smi`
+nvmm() {
```

Comment:
Not used in this PR, but I use this now ~routinely in local dev / testing; and I can see that we will use it in the test suite, too.